wgpu-core/src/device/queue.rs

```diff
@@ -483,6 +482,10 @@ impl Global {
             .and_then(Arc::into_inner)
             .ok_or_else(|| QueueWriteError::Transfer(TransferError::InvalidBufferId(buffer_id)))?;
 
+        // TODO: bubble up safety requirements, they currently hold but we should make functions
```

Comment:
question(blocking): Should this be resolved before merging?

Reply:
I don't intend to resolve this in this PR. After degenerification and removal of registries we might not even need to resolve it since we can make the API safe by not giving users the pointer at all.

Reply:
Ah, shall we file an issue, then, and remove this comment? If we don't want an issue to track it, then I wonder if we should even include the comment. 

Will downgrade to non-blocking with a response to the above.